dist/common/systemd/scylla-debugfs.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Scylla debugfs configrator
+After=local-fs.target
```

Comment:
instead of starting this service after `local-fs` is fully ready, i'd suggest add procfs and debugfs directories to `RequiresMountsFor=`, see https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#RequiresMountsFor=